### PR TITLE
[14.0][MIG] hr_attendance_geolocation: Migration to 14.0

### PR DESCRIPTION
**Pull request by MeetKD**
Standard Migration.

### CODE REVIEW

File: hr_attendance_geolocation/views/hr_attendance_views.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_attendance_view_form" model="ir.ui.view">
```
Comment (olgamarcocb) [inline hr_attendance_geolocation/views/hr_attendance_views.xml:3]: The form is not opened on 14.0, as it is an editable tree. Can you check it please

**Review by olgamarcocb — COMMENTED** [review]
The forms were not working. In order to check on runbot, you need to use https

File: hr_attendance_geolocation/views/hr_attendance_views.xml (hunk 1 above)
Comment (MeetKD) [reply hr_attendance_geolocation/views/hr_attendance_views.xml:3]: You can switch to kanban view and click on the kanban card to check the form details.

Comment (olgamarcocb) [reply hr_attendance_geolocation/views/hr_attendance_views.xml:3]: Great! You're right but it would be interesting if it could be opened on 14.0

**Review by olgamarcocb — APPROVED** [review]
Approved but check the request change :+1:

**Review by dsolanki-initos — APPROVED** [review]
LGTM

**Comment by dsolanki-initos** [comment]
Can we merge this PR?

**Comment by hkapatel-initos** [comment]
Hello maintainers,
It would be nice if you are trying to merge it

**Comment by hkapatel-initos** [comment]
Can we merge this PR?

**Comment by hkapatel-initos** [comment]
Can we merge this PR?

**Comment by hkapatel-initos** [comment]
Hello maintainers,
It would be nice if you are trying to merge it

**Comment by dsolanki-initos** [comment]
It would be nice if maitainers could merge this PR

**Comment by dsolanki-initos** [comment]
It would be nice if maintainers could merge this PR

**Comment by dsolanki-initos** [comment]
It would be nice if maintainers could merge this PR

**Comment by dsolanki-initos** [comment]
@pedrobaeza Can we merge this?

**Pull request merged**